pkg/skaffold/generate_pipeline/generate_pipeline.go

```diff
@@ -139,7 +139,33 @@ func generateBuildTask(buildConfig latest.BuildConfig) (*tekton.Task, error) {
 		},
 	}
 
-	return pipeline.NewTask("skaffold-build", resources, steps), nil
+	var volumes []corev1.Volume
+	if buildConfig.Artifacts[0].KanikoArtifact != nil {
+		volumes = []corev1.Volume{
+			{
+				Name: "kaniko-secret",
```

Comment:
lets use constants everywhere for kaniko.
Define constant in the local pkg scope.